packages/terra-site/package.json

```diff
@@ -26,10 +26,11 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "moment": "^2.17.1",
-    "react": "15.4.2",
-    "react-dom": "15.4.2",
-    "react-intl": "2.2.3",
-    "react-router": "^3.0.2",
+    "prop-types": "^15.5.8",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
In this specific case, it doesn't matter too much. Using the [caret operator](https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/#giving-npm-permission-to-install-newer-version) with `^15.5.4` or `^15.4.2` will result npm installing the latest released `15.x` version of react and react-dom.